lib/cylc/cfgspec/globalcfg.py

```diff
@@ -45,7 +45,7 @@
 coercers['interval_seconds'] = (
     lambda *args: coerce_interval(*args, check_syntax_version=False))
 coercers['interval_minutes'] = lambda *a: coerce_interval(
-    *a, back_comp_unit_factor=60)
+    *a, back_comp_unit_factor=60, check_syntax_version=False)
```

Comment:
Any chance of `*args` rather than `*a` for style consistency with the other two coercers?